components/Radio/__tests__/Radio.test.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react"
+import RadioGroup from "../index"
+import { render, cleanup, fireEvent } from "react-testing-library"
+import "jest-styled-components"
+import "jest-dom/extend-expect"
+
+afterEach(cleanup)
+
+describe("Radio", () => {
+  describe("renders radio correctly", () => {
+    it("Small with 2 in group", () => {
+      const { asFragment } = render(
+        <RadioGroup size="small">
+          <RadioGroup.Item value={1}>Option 1</RadioGroup.Item>
+          <RadioGroup.Item value={2}>Option 2</RadioGroup.Item>
+        </RadioGroup>
+      )
+      expect(asFragment()).toMatchSnapshot()
+    })
+
+    it("Small with 2 in group with disabled", () => {
+      const { asFragment } = render(
+        <RadioGroup size="small" disabled defaultValue={1}>
+          <RadioGroup.Item value={1}>Option 1</RadioGroup.Item>
+          <RadioGroup.Item value={2}>Option 2</RadioGroup.Item>
+        </RadioGroup>
+      )
+      expect(asFragment()).toMatchSnapshot()
+    })
+
+    it("Medium with 2 in a group", () => {
+      const { asFragment } = render(
+        <RadioGroup size="small" disabled defaultValue={1}>
+          <RadioGroup.Item value={1}>Option 1</RadioGroup.Item>
+          <RadioGroup.Item value={2}>Option 2</RadioGroup.Item>
+        </RadioGroup>
+      )
+      expect(asFragment()).toMatchSnapshot()
+    })
+
+    it("renders with single child correctly", () => {
```

Comment:
From the figma link, it looks like there should be a minimum of two values in the group. Have we thought of enforcing that condition?

Reply:
eeks no, do we throw and err or just print a warning or do not render radios at all when <2?

Reply:
For now did a warning

Reply:
@anshumanv did you check if it is possible through typescript?

Reply:
Can change the type to `ReactNode[]` in Group to only accept array of children which will be always >=2 but then no errs thrown on `<RadioGroup />`